auction-house/program/src/execute_sale/mod.rs

```diff
@@ -34,6 +39,7 @@ pub struct ExecuteSale<'info> {
 
     /// CHECK: Validated in execute_sale_logic.
     /// Metaplex metadata account decorating SPL mint account.
+    //@TODO: re-enable this later #[account(mut)]
```

Comment:
Is this meant to be commented out?

Reply:
Yup, this is the breaking change we'll add back in later.